docs/getting-started-guides/scratch.md

```diff
@@ -342,6 +354,7 @@ so that kube-proxy can manage iptables instead of docker.
 
 You may want to increase the number of open files for docker:
    - `DOCKER_NOFILE=1000000`
+Where this config goes depends on your node OS.  For example, GCE's Debian-based distro uses `/etc/default/docker`.
```

Comment:
Do you want an extra newline before this line? This is formatted on the same line as - `DOCKER_NOFILE=...` but applies to the entire docker config section.